src/credentials/auth.ts

```diff
@@ -301,6 +301,8 @@ interface ConnectionStateChangeEvent {
     readonly state: ProfileMetadata['connectionState']
 }
 
+export const invalidConnection = 'InvalidConnection'
```

Comment:
should this live in errors.ts ?

Reply:
Should I make an Enum in `errors.ts`?

```
enum ErrorCodes {
    InvalidConnection = 'InvalidConnection'
}
```

Reply:
sure, though enums are never needed (essentially deprecated) in typescript, see https://www.executeprogram.com/blog/typescript-features-to-avoid